solana/pyth2wormhole/client/src/main.rs

```diff
@@ -173,12 +173,13 @@ async fn main() -> Result<(), ErrBox> {
             daemon,
         } => {
             // Load the attestation config yaml
-            let attestation_cfg: AttestationConfig =
+            let mut attestation_cfg: AttestationConfig =
```

Comment:
maybe this one is not necessary here 🤔

Reply:
It's not possible to take a mutable reference to an immutable variable